### PR TITLE
fix(bugs): Fix Android dark mode and max-width on titles

### PR DESCRIPTION
**Pull request by damassi**
Fixes: 
- [MOPLAT-627] - Safe areas should be white (in non-dark mode) on Android
- [MOPLAT-663] - Cap max-width and add elipses on title when select button is visible 

https://user-images.githubusercontent.com/236943/224841324-5b49bfb6-8b5d-46e1-b6a7-f6f472ebe2d7.mp4

Screenshot 2023-03-13 at 2 58 30 PM

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.

[MOPLAT-627]: https://artsyproduct.atlassian.net/browse/MOPLAT-627?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MOPLAT-663]: https://artsyproduct.atlassian.net/browse/MOPLAT-663?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Providers.tsx [hunk 1]
```diff
@@ -51,7 +51,10 @@ const ThemeProvider: React.FC = ({ children }) => {
     <Theme theme={theme}>
       {children}
 
-      <StatusBar barStyle={isDarkMode ? "light-content" : "dark-content"} />
+      <StatusBar
+        barStyle={isDarkMode ? "light-content" : "dark-content"}
+        backgroundColor={isDarkMode ? "black" : "white"}
```
Comment (damassi) [inline src/Providers.tsx:56]: android needs this, as barStyle is only for iOS

File: src/components/Screen/notExposed/ActualHeader.tsx [hunk 2]
```diff
@@ -89,24 +94,26 @@ export const ActualHeader = ({
...
-                    justifyContent: "flex-start",
-                    flex: 1,
-                  }}
-                >
-                  {actualTitle}
-                </Animated.View>
-              )
-            : actualTitle)}
-      </Flex>
+      {!isSelectModeActive && (
```
Comment (damassi) [inline src/components/Screen/notExposed/ActualHeader.tsx:97]: when a user has clicked the select button, it activates select mode; at that time we hide the title to display the "Select All" button on the left.

File: src/components/Screen/notExposed/ActualHeader.tsx [hunk 3]
```diff
@@ -89,24 +94,26 @@ export const ActualHeader = ({
...
+                  <Animated.View
+                    entering={FadeInLeft}
+                    exiting={FadeOutLeft}
+                    style={{
+                      alignItems: "flex-start",
+                      justifyContent: "flex-start",
+                      flex: 1,
+                    }}
+                  >
+                    <Flex width={isTablet() ? "100%" : "70%"}>{actualTitle}</Flex>
```
Comment (damassi) [inline src/components/Screen/notExposed/ActualHeader.tsx:111]: /shrug. I'm sure we could do some measuring, etc, but lets pass this one on 😄

**Review by brainbicycle — APPROVED** [review]
Looking great!

**Pull request merged**